firebase/firebase-handler.js

```diff
@@ -81,6 +81,29 @@ class FirebaseHandler {
     }
 
     checkFeatureExists(feature) {
+        const subCheck = (newValue, parentKey, changedFeatures, dbFeature) => {
+            if (typeof newValue == "object") {
```

Comment:
we put the optional transform object in fileInfo but it looks like this is intended for features

Reply:
i was just fixing a problem here with the way the check was working. this isn't related to the transform object